### PR TITLE
Feature/nft tags backend

### PR DESCRIPTION
**Pull request by juliswer**

### CODE REVIEW

File: src/items/repositories/item.repository.ts [hunk 1]
```diff
@@ -53,14 +57,60 @@ export class ItemRepository extends Repository<Item> {
   }
 
   async findByAccount(accountId: string): Promise<Item[]> {
-    return this.getItemQueryBuilder()
+    return this.createQueryBuilder('item')
+      .select([
```
Comment (mmiranda-stackit) [inline src/items/repositories/item.repository.ts]: Whay aren't you using the querybuilder? that way we will reuse a ton of repeated code

File: src/items/repositories/item.repository.ts [hunk 2]
```diff
@@ -53,14 +57,60 @@ export class ItemRepository extends Repository<Item> {
...
+      .innerJoin('item.tags', 'tag')
       .where('item.ownerId = :accountId OR item.authorId = :accountId', { accountId })
       .orderBy('item.createdAt', 'DESC')
       .getMany();
   }
 
   async findById(itemId: string): Promise<Item> {
-    return this.getItemQueryBuilder()
+    return this.createQueryBuilder('item')
+      .select([
```
Comment (mmiranda-stackit) [inline src/items/repositories/item.repository.ts]: same here

File: src/main.ts [hunk 3]
```diff
@@ -15,7 +15,7 @@ async function bootstrap() {
   const adapterHost = app.get(HttpAdapterHost);
   app.useGlobalFilters(new GlobalExceptionFilter(adapterHost));
 
-  const port = process.env.PORT || 3333;
+  const port = process.env.PORT || 3334;
```
Comment (mmiranda-stackit) [inline src/main.ts:18]: Don't modify this, I don't think is part of the task

**Review by mmiranda-stackit — CHANGES_REQUESTED** [review]

File: src/main.ts (hunk 3 above)
Comment (juliswer) [reply src/main.ts:18]: That's because of my Local Env. Throw me an error if I use the 3333 port

Comment (mmiranda-stackit) [reply src/main.ts:18]: If the problem is local, so you should not update anything in the server

**Review by mmiranda-stackit — APPROVED** [review]

**Pull request merged**